Server/src/main/java/org/openas2/cmd/processor/RestCommandProcessor.java

```diff
@@ -88,30 +86,24 @@ public void init(Session session, Map<String, String> parameters) throws OpenAS2
             logger.info(this.getName() + " initialized...");
             // create a resource config that scans for JAX-RS resources and providers
             final ResourceConfig rc = new ResourceConfig();
-            rc.register(new LoggerRequestFilter(logger))
-                .register(new AuthenticationRequestFilter(
-                    parameters.getOrDefault("userid","userid"),
-                    parameters.getOrDefault("password","pWd")
-                ))
-                .register(new ApiResource(this))
-                .register(new CORSFilter());
+            rc.register(new LoggerRequestFilter(logger)).register(new AuthenticationRequestFilter(parameters.getOrDefault("userid", "userid"), parameters.getOrDefault("password", "pWd"))).register(new ApiResource(this)).register(new CORSFilter());
```

Comment:
I think splitting fluent API calls into multiple lines makes it more readable

Reply:
This is ok but for the automated reformat this is probably a manual effort so leave it out if it is and someone can manually change later when working on the file.

Reply:
Intellij can handle this. I will reformat.